tests/etl/test_perf_data_adapters.py

```diff
@@ -415,8 +415,7 @@ def test_alert_generation(test_repository,
                           max_back_window=extra_suite_metadata.get('maxBackWindow'),
                           fore_window=extra_suite_metadata.get('foreWindow'))
 
-    expected_num_alerts = len(filter(lambda x: x is True, [expected_suite_alert,
-                                                           expected_subtest_alert]))
+    expected_num_alerts = sum([expected_suite_alert, expected_subtest_alert])
```

Comment:
Yeah me neither; `True` coerces to `1`:
https://stackoverflow.com/a/12766000

I almost used `sum(1 for x in [expected_suite_alert, expected_subtest_alert] if x)` since it makes it marginally clearer what is happening, but I think from context / variable names it should be clear enough as is.